clusterloader2/README.md

```diff
@@ -1,21 +1,81 @@
 # ClusterLoader
 
-### Running ClusterLoader
+## Running ClusterLoader
 
 To run ClusterLoader type:
 ```
-go run cmd/clusterloader.go --kubeconfig=kubeConfig.yaml --testconfig=config.json
+go run cmd/clusterloader.go --kubeconfig=kubeConfig.yaml --testconfig=config.yaml
 ```
-Flags kubeconfig and testconfig are necessary.
+Flags kubeconfig, testconfig and nodes are necessary.
 
-#### Flags
+### Flags
 
+#### Required
+
+These flags are required for any test to be run.
  - kubeconfig - path to the kubeconfig file.
- - testconfig - path to the test config file.
+ - testconfig - path to the test config file. This flag can be used multiple times
+if more than one test should be run.
 
-### Vendor
+#### Optional
 
-Vendor is created using [govendor].
+ - nodes - number of nodes in the cluster.
+If not provided, test will assign the number of available schedulable cluster nodes.
+ - report-dir - path to directory, where summaries files should be stored.
+If not specified, summaries are printed to standard log.
+
+## Tests
+
+### Test definition
+
+Tests can be written in yaml or json. They need to be parsable to this [api]. Test definition supports templating.
+ The provided variable that can be used is ```{{.Nodes}}``` which is a number of nodes in the cluster. \
+Test is composed of steps, which define the number and definition of required objects.
+Each object definitions has path to object temaplate defined. To each object parameters can be passed using ```templateFillMap``` map,
+which is part of the object definition. \
+Design document for the test definition can be found here: [design doc]. \
```

Comment:
The orgnization in this section is quite chaotic in my opinion. It should be more like:

Test definition is instantiation of this [api] (in json or yaml). The motivation and description of the API can be find in [design doc].
Definitions of test as well as definitions of individual objects support templating, with some predefined value (e.g. {{.Nodes}} and ability to pass values via templateFillMap.


What i wrote is not ideal, but it's at least not randomly mentioning different topics.

Reply:
I rewrote this section based on your suggestion.